packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx

```diff
@@ -71,8 +70,13 @@ const RenderTmpl: FC<{
   return (
     <StructRender
       struct={tmpl.struct}
-      options={{ ...options, autoScaleMargin: 15 }}
       {...props}
+      options={{
+        ...options,
+        autoScaleMargin: 10,
+        cachePrefix: 'templates',
+        downScale: true
```

Comment:
why not name it starting with 'is' so that it's clear it's a boolean?

Reply:
This one I did via analogy to "autoScale" property that we already had.